tests/MoBi.Tests/Presentation/EditParameterListSpecs.cs

```diff
@@ -375,4 +385,67 @@ public void should_tell_quantuity_task_to_set_vlaue()
          A.CallTo(() => _favoriteTask.SetParameterFavorite(_parameter, true)).MustHaveHappened();
       }
    }
+
+   public class When_container_does_not_have_a_name : concern_for_EditParameterListPresenter
+   {
+      protected IContainer _container;
+      private readonly string _containerType = "Container";
+
+      protected override void Context()
+      {
+         base.Context();
+         _container = new Container();
+         _view = new EditParametersInContainerView(A.Fake<IToolTipCreator>(), A.Fake<ValueOriginBinder<ParameterDTO>>());
+         sut = new EditParametersInContainerPresenter(_view, _formulaMapper, _parameterMapper, _inteactionTasks,
+            _distributeParameterPresenter, _parameterPresenter, _quantityTask, _interactionTaskContext, _clipboardManager, _editTask, _selectReferencePresenterFactory, _favoriteTask, _typeResolver);
```

Comment:
I think this part should be unnecessary. I know that it's because you want to check the view for ParentProperty, which has no getter. That's tricky. I pulled the branch and checked how it could be done using FakeItEasy fakes.

```
A.CallTo(_view).Where(x => x.Method.Name.Equals("set_ParentName") && x.Arguments.Get<string>(0).Equals("Something")).MustHaveHappened();
```

Reply:
well done!